[14.0][MIG] product_multi_price

watthanun opened the pull request.

chienandalu committed:
[ADD] product_multi_price: New Module

pedrobaeza committed:
[IMP] product_multi_price: Several improvements

* Adjusted terminology
* Better positioning of the prices list
* Adjusted README
* Remove useless constraint
* Make multi price name translatable
* Apply some guidelines
* Remove compute proxy field as not finally used, and thus the ORM cache, but
  needed then to override initial computation.

chienandalu committed:
[FIX] product_multi_price: portal and public user error

When a not internal user (portal/public) with a pricelist that
contains multi price rules tries to see a page with a product affected
for such rules, an exception raises due to lack of permissions over
product.multi.price model.

pedrobaeza committed:
[FIX] product_multi_price: Widget selection doesn't work inside tree view

We mimick it putting regular widget with options no_open and no_create.

chienandalu committed:
[MIG] product_multi_price: Migration to 12.0

Commit:
[IMP] product_multi_price: add show multi-price access group.

eLBati committed:
Added translation using Weblate (Italian)

eLBati committed:
Translated using Weblate (Italian)

Currently translated at 73.5% (25 of 34 strings)

Translation: product-attribute-12.0/product-attribute-12.0-product_multi_price
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-12-0/product-attribute-12-0-product_multi_price/it/

claudiagn committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (34 of 34 strings)

Translation: product-attribute-12.0/product-attribute-12.0-product_multi_price
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-12-0/product-attribute-12-0-product_multi_price/es/

qgroulard committed:
[IMP] product_multi_price: black, isort, prettier

qgroulard committed:
[MIG] product_multi_price: Migration to 13.0

oca-travis committed:
[UPD] Update product_multi_price.pot

OCA-git-bot committed:
[UPD] README.rst

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-13.0/product-attribute-13.0-product_multi_price
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-13-0/product-attribute-13-0-product_multi_price/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: product-attribute-13.0/product-attribute-13.0-product_multi_price
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-13-0/product-attribute-13-0-product_multi_price/

rousseldenis commented on product_restricted_type/__manifest__.py:
```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Product Restricted Type",
+    "version": "14.0.1.0.0",
+    "author": "ForgeFlow," "Odoo Community Association (OCA)",
```
You can remove the " ", just let a space

claudiagn committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (28 of 28 strings)

Translation: product-attribute-13.0/product-attribute-13.0-product_multi_price
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-13-0/product-attribute-13-0-product_multi_price/es/

claudiagn committed:
Added translation using Weblate (Catalan)

claudiagn committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (28 of 28 strings)

Translation: product-attribute-13.0/product-attribute-13.0-product_multi_price
Translate-URL: https://translation.odoo-community.org/projects/product-attribute-13-0/product-attribute-13-0-product_multi_price/ca/

newtratip committed:
[IMP] product_multi_price: black, isort, prettier

newtratip committed:
[MIG] product_multi_price: Migration to 14.0
